README.md

```diff
@@ -1,71 +1,92 @@
 
 ## Importing Data into MapR JSON Tables
 
 The application use the Yelp Academy Dataset that contains a list of Business and User Reviews.
 
-#### 1- Download the Yelp Dataset from [here](https://www.yelp.com/dataset_challenge)
+#### 1. Download the Yelp Dataset from [here](https://www.yelp.com/dataset_challenge)
 
-#### 2- Copy the dataset to MapR Cluster
+#### 2. Unarchive the dataset
 
 ```
-scp yelp_dataset_challenge_round9.tar mapr@mapr60:/mapr/my.cluster.com/tmp/
+$ tar xvf yelp_dataset_challenge_round9.tar
 ```
 
-* `mapr60` is one of the node of your cluster
-* `my.cluster.com` is the name of your cluster
+#### 3. Copy the dataset to MapR Cluster
+
+##### 3a. Copy data to the node 
+To do this, we first copy the file (say `business.json`) to the cluster node.
+```
+$ cd dataset
 
-#### 3- Unarchive the dataset
+$ scp business.json root@<hostname>:/tmp/
 
-Connect to one of the node of your cluster using `mapr` user
+$ scp review.json root@<hostname>:/tmp/
 
+$ scp user.json root@<hostname>:/tmp/
 ```
-$ cd /mapr/my.cluster.com/tmp/
 
-$ tar xvf yelp_dataset_challenge_round9.tar
+If you are working with Developer Sandbox container, you copy the file over to container.
 ```
+$ docker cp business.json <container-id>:/tmp/
 
-#### 4- Import the data into MapR DB Tables
-
-Create new tables using maprcli tool
+$ docker cp review.json <container-id>:/tmp/
 
+$ docker cp user.json <container-id>:/tmp/
 ```
-$ maprcli table create -path /apps/business -tabletype json
+Similarly, we can copy the rest of files or copy the entire `dataset` directory.
 
-$ maprcli table create -path /apps/review -tabletype json
+##### 3b. Copy data in to MapR-XD
 
-$ maprcli table create -path /apps/user -tabletype json  
+There are two ways to put data in to MapR-XD
+
+- By simple copy if [NFS service is installed](https://maprdocs.mapr.com/home/AdministratorGuide/AccessDataWithNFS.html) and the cluster is mounted at `/mapr`.
+```
+$ cp business.json /mapr/<cluster-name>/tmp/
 ```
 
+When using developer sandbox container,
+```
+$ cp business.json /mapr/maprdemo.mapr.io/tmp/
+```
 
+- By running hadoop commands to put data.
+```
+hadoop fs -put business.json /tmp/
+```
 
-#### 5- Import the JSON documents into MapR tables
+#### 4. Import the JSON documents into MapR-DB JSON tables
 
 ```
 
-$ mapr importJSON -idField business_id -src /tmp/yelp_academic_dataset_business.json -dst /apps/business -mapreduce false
+$ mapr importJSON -idField business_id -src /tmp/business.json -dst /apps/business -mapreduce false
 
-$ mapr importJSON -idField review_id -src /tmp/yelp_academic_dataset_review.json -dst /apps/review -mapreduce false
+$ mapr importJSON -idField review_id -src /tmp/review.json -dst /apps/review -mapreduce false
 
-$ mapr importJSON -idField user_id -src /tmp/yelp_academic_dataset_user.json -dst /apps/user -mapreduce false
+$ mapr importJSON -idField user_id -src /tmp/user.json -dst /apps/user -mapreduce false
 
 ```
 
```

Comment:
Just noticed this one -- not directly your changes.  The sentence "You have now 3 JSON tables ...." should be "You now have 3 JSON tables ..."